react.md

```diff
@@ -5,40 +5,46 @@ Answer the following questions. First, without external resources. Challenge you
 
 1a. Indicate which of the following statements about React are false:
 
-- React is a modern, efficient answer to complex UI applications (true/false)
-- React will only render on the server using Node.js (true/false)
-- React is a full stack framework for modern web applications (true/false)
-- React is a flexible library that plays the role of V in an MVC framework (true/false)
-- You should always update a component's state directly using this.state (true/false)
-- React is made up of encapsulated components that manage their own state (true/false)
-- React components render HTML (true/false)
+- React is a modern, efficient answer to complex UI applications (true/false) true
+- React will only render on the server using Node.js (true/false) false <https://daveceddia.com/do-i-need-nodejs-backend-for-react-angular/>
+- React is a full stack framework for modern web applications (true/false) false
```

Comment:
React is just a frontend language. It needs a backend language to make a full stack app.